controllers/clusters/kafka_controller.go

```diff
@@ -172,76 +172,6 @@ func (r *KafkaReconciler) startJobs(k *v1beta1.Kafka) error {
 	return nil
 }
 
-func (r *KafkaReconciler) handleOnPremisesCreation(ctx context.Context, k *v1beta1.Kafka, l logr.Logger) error {
-	instaModel, err := r.API.GetKafka(k.Status.ID)
-	if err != nil {
-		l.Error(err,
-			"Cannot get cluster from the Instaclustr API",
-			"cluster name", k.Spec.Name,
-			"data centres", k.Spec.DataCentres,
-			"cluster ID", k.Status.ID,
-		)
-		r.EventRecorder.Eventf(
-			k, models.Warning, models.FetchFailed,
-			"Cluster fetch from the Instaclustr API is failed. Reason: %v",
-			err,
-		)
-		return err
-	}
-
-	iKafka := v1beta1.Kafka{}
-	iKafka.FromInstAPI(instaModel)
-
-	bootstrap := newOnPremisesBootstrap(
-		r.Client,
-		k,
-		r.EventRecorder,
-		iKafka.Status.ToOnPremises(),
```

Comment:
It looks like `ToOnPremises` constructors may be deleted too

Reply:
Let's keep them for the future phases